### ISSUE TITLE
Turn on and off hot water boiler

### ISSUE THREAD

**Issue by geurti64** [robertklep/nl.nefit.easy#6]
It would be nice to turn on and off the hot water boiler in the Central Heating system.

**Comment by robertklep** [comment]
Looking into this, but I don't know which [endpoint](https://github.com/marcelrv/nefit_easy_protocol) I need to use for this :(

**Comment by robertklep** [comment]
@geurti64 I do need help, but I realize that it might be difficult to provide.

Like I said, I don't have a Nefit CV with a boiler. I can turn on/off the hot water supply for my device, but I don't think that's the same as switching off the boiler (to keep the boiler at temperature), because turning off the hot water supply would disable outputting hot water entirely.

So basically, I have no idea how to implement this feature. The problem is that unless you're familiar with the internals of the Nefit Easy, it's very difficult to find out how to disable the boiler programmatically. So for the time being, I think that I simply won't be able to implement this feature.

**Comment by robertklep** [comment]
@itshows you're correct, if you can show me how to toggle the hot water supply, I can implement it 👍🏻

**Comment by itshows** [comment]
what do you mean exactly?
the cli command that does the job is: easy [options] hot-water-supply [ on | off ]
after that command the hot water icon on the thermostat toggles. So if that function can be built in into the homey app that would be perfect.

**Comment by robertklep** [comment]
@itshows oh, sorry for the confusion: that setting turns off the ability for the Nefit to provide hot water.

However, we're looking for a setting to turn of the _boiler_. I don't think that's the same thing (but I could be mistaken).

**Comment by itshows** [comment]
@robertklep That's right. That toggle function (hot-water-supply) would (for me anyway) be very welcome on the homey app.
Where in the nefit easy thermostat app is the ability to turn off the boiler itself. I have been looking in the app, but cannot find that option.

**Comment by robertklep** [comment]
To be honest, I have no idea (I don't have a boiler myself).

I will add a new TODO to implement toggling the hot water supply from Homey :)

**Comment by itshows** [comment]
That would be very nice and very much appreciated.
I do have a boiler, but cannot find that option in the app. 
I believe the only possibility is to toggle the hot water supply which turns off or on the hot water in total.
Thanks. If there is anything I can do please let me know. 👍🏻

**Comment by geurti64** [comment]
"Where in the nefit easy thermostat app is the ability to turn off the boiler itself. I have been looking in the app, but cannot find that option."

@Itshows, have you find this option?
Just hit the timer function and then hit the hot water tap on or off.

@Robert Do you think it's still possible?

**Comment by robertklep** [comment]
@geurti64 theoretically it should be possible, but I still don't know how to implement this :(
